source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacyExtract.sql

```diff
@@ -212,7 +212,7 @@ BEGIN TRY
           ,[ShortCourse]
           ,[InPatient]
           ,[Comments]
-          ,[OtherComments]
+          ,LEFT([OtherComments], 60) AS ImmunosuppressionComments
```

Comment:
Why are we using 60 here, but 100 below?

Reply:
Yes, I'm not sure there is a reason - will update above to 100.